src/pages/Environments.js

```diff
@@ -219,8 +233,8 @@ const Environments = () => {
     ])
   }
   const renderWorkspaceForApps = app => {
-    const { appType, labels: { saturnWorkspaceNamespace, saturnWorkspaceName } } = app
-    const multipleApps = workspaceHasMultipleApps(appsByProject[app.googleProject], appType)
+    const { appType, googleProject, labels: { saturnWorkspaceNamespace, saturnWorkspaceName } } = app
```

Comment:
Instead of all of this, could you just do this?
```suggestion
    const { appType, googleProject, labels: { saturnWorkspaceNamespace = googleProject, saturnWorkspaceName } } = app
```

Reply:
Yes! I didn't know we could use a parameter deconstructed on the same line right away like that 🤩 

There's still the issue of having to do that in 3 different places but I don't think avoiding that is worth all the previous complexity.